.github/workflows/deploy.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches: # White-list of deployable tags and branches. Note that all white-listed branches cannot include any `/` characters
       - next
+      - SPARK-455628-Pipeline-changes-to-push-to-documentation-branch
```

Comment:
This line should be removed.

Adding this comment as a placeholder. We have to merge this PR only after resolving this comment.

Reply:
This is being done only for testing purpose, will be removed